### DISCUSSION TITLE
Usage terms for StreetComplete logo (app icon)

### DISCUSSION THREAD

**Discussion by das-g** [streetcomplete/StreetComplete#4229] [Q&A] [answered]
Where does the StreetComplete logo / app icon come / derive from?

<details><summary>variants of StreetComplete app icon (click to show)</summary>

![](https://raw.githubusercontent.com/streetcomplete/StreetComplete/0b292ae71aeb5d95e0c316d26f323d3c0b5b4cef/res/graphics/app%20icon/appicon_flat.svg) ![](https://raw.githubusercontent.com/streetcomplete/StreetComplete/0b292ae71aeb5d95e0c316d26f323d3c0b5b4cef/res/graphics/app%20icon/appicon_fancy-rect.svg) ![](https://raw.githubusercontent.com/streetcomplete/StreetComplete/0b292ae71aeb5d95e0c316d26f323d3c0b5b4cef/res/graphics/app%20icon/appicon_fancy.svg)

</details>

Was is created specifically for StreetComplete? If so, can it be freely used by other parties, also for purposes other than referring to StreetComplete?

If it's only to be used by StreetComplete and/or for referring to StreetComplete, you may want to contact [Lenzin Informatik](https://www.lenzin-informatik.ch/) about that, as they use an eerily similar icon on their website on page https://www.lenzin-informatik.ch/website-konzept

<details><summary>Screenshot of https://www.lenzin-informatik.ch/website-konzept (click to show)</summary>

![image](https://user-images.githubusercontent.com/97746/180433230-ea4e8fae-0df2-452c-9b75-b4ec9e456b66.png)

</details>
<details><summary>icon on https://www.lenzin-informatik.ch/fileadmin/Bilder/Icons/crossroads.svg (click to show)</summary>

![](https://www.lenzin-informatik.ch/fileadmin/Bilder/Icons/crossroads.svg)

</details>

(Found via https://www.openstreetmap.org/note/3276658 mentioned [here](https://matrix.to/#/!SKrlcOagVFDvtqOndy:matrix.org/$1658430542206243PrckA:matrix.org?via=matrix.org&via=binarylogic.ch&via=matrix.wirebug.ch) in `#osm-ch`.)

**Comment by matkoniecz** [comment, marked as answer]
See https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/authors.txt#L36

They likely use https://www.elegantthemes.com/blog/freebie-of-the-week/beautiful-flat-icons-for-free licensed GPL 2.0 - like SC

**Reply by das-g** [reply]
> See https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/authors.txt#L36

Oops, I didn't notice that file. (Probably due to the many directories filling the screen.) Thanks for pointing to it.

> They likely use https://www.elegantthemes.com/blog/freebie-of-the-week/beautiful-flat-icons-for-free licensed GPL 2.0 - like SC

That'd be it, indeed. :+1:
